Jenkinsfile

```diff
@@ -0,0 +1,3 @@
+// Builds a module using https://github.com/jenkins-infra/pipeline-library
+// TODO: Explicitly require LTS baseline once SSHD Module 2.x gets into LTS
```

Comment:
I do. My plan it to test...

* Minimal supported LTS version by the module
* Latest supported LTS version 
* Latest weekly
* Maybe other particular LTS lines I am interested in